packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -144,23 +144,19 @@ function BlockPatternsTabs( {
 
 	return (
 		<>
+			<PatternInserterPanel
+				selectedCategory={ patternCategory }
+				patternCategories={ populatedCategories }
+				onClickCategory={ onClickCategory }
+				openPatternExplorer={ () => setShowPatternsExplorer( true ) }
+			/>
 			{ ! showPatternsExplorer && (
-				<>
-					<PatternInserterPanel
-						selectedCategory={ patternCategory }
-						patternCategories={ populatedCategories }
-						onClickCategory={ onClickCategory }
-						openPatternExplorer={ () =>
-							setShowPatternsExplorer( true )
-						}
-					/>
-					<BlockPatternsCategory
-						rootClientId={ rootClientId }
-						onInsert={ onInsert }
-						selectedCategory={ patternCategory }
-						populatedCategories={ populatedCategories }
-					/>
-				</>
+				<BlockPatternsCategory
+					rootClientId={ rootClientId }
+					onInsert={ onInsert }
+					selectedCategory={ patternCategory }
+					populatedCategories={ populatedCategories }
+				/>
```

Comment:
Any reason we conditionally render this based on `showPatternsExplorer`?

Reply:
For performance reasons. If we open the explorer there is no need to continue work on rendering the patterns there, as it's a heavy operation. This makes sense though if we open the explorer before all patterns have rendered in the inserter, which can be the case many times.

Reply:
But if patterns are rendered, and we open/close the modal, they will re-rendered.

It made things a little slow when I was testing the fix.